Readme.md

```diff
@@ -2,10 +2,14 @@ A tiny utility to post current directory contents to http://plnkr.co/edit
 
 ## Anonymous Usage
 
+0. Add `bin` project folder to your path
 1. Go the the dir you want to plunk.
 2. Run `plunk`
 
-Or `plunk --dir path/to/dir --desc "A string describing the plunk" --tags "comma,separated,tags"`.
+Or `plunk --dir path/to/dir --desc "A string describing the plunk" --tags "comma,separated,tags" --glob "pattern"`.
+Specify default parameters in `defaults.json`.
```

Comment:
defaults.json is in the app root. I guess the line about `defaults.json` can be removed as confusing.